plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -698,6 +698,7 @@ void GSHacksDlg::OnInit()
 	CheckDlgButton(m_hWnd, IDC_PRELOAD_GS, theApp.GetConfigB("preload_frame_with_gs_data"));
 	CheckDlgButton(m_hWnd, IDC_ALIGN_SPRITE, theApp.GetConfigB("UserHacks_align_sprite_X"));
 	CheckDlgButton(m_hWnd, IDC_TC_DEPTH, theApp.GetConfigB("UserHacks_DisableDepthSupport"));
+	CheckDlgButton(m_hWnd, IDC_8BITCPUFBHACK, theApp.GetConfigB("UserHack_cpu_8bis_fb_conversion"));
```

Comment:
Ditto. 

Also, the label should read `"UserHacks_CPU_8Bit_FB_Conversion"` or something similar to match the common format here.